### DISCUSSION TITLE
Shortcut for "Save/Load Reading Progress Sync" ?

### DISCUSSION THREAD

**Discussion by viper73** [koreader/koreader#12672] [Ideas]
Is it possible to make an Shortcut for the "save reading Progress" and "load reading Progress" Buttons?

It is a long way to this buttons every time, so it would be more practical to have them directly in an new Menu on the top bar like "Progress Sync" instead of going to the "Tools" Button, then in the "Reading Progress" Button, and then to the "Save/Load" Button.

Or maybe this is an idea for an upcoming update for KOReader.

**Comment by hius07** [comment]
You can assign gestures, the actions are under the Reader section in the Gesture manager.
Or make a profile with all actions you need, and call this profile with a suitable gesture.

**Reply by viper73** [reply]
Both working great, thank you!
